scripts/ghidra/SyncBromaScript.java

```diff
@@ -681,8 +681,9 @@ private void handleImportMembers() throws Exception {
                             padLength = mem.paddings.get(args.platform);
                         }
                         else {
-                            wrapper.printfmt("Warning: class {0} contains unimplemented padding on this platform", fullName);
-                            padLength = 1;
+                            // I think there should be no warning, because sometimes paddings are exclusive to specific operation systems
+                            // wrapper.printfmt("Warning: class {0} contains unimplemented padding on this platform", fullName);
+                            padLength = 0;

```

Comment:
This is a bad idea. The `padLength` is intentionally set to one so the member exporting script knows there was a padding there to export, since it replaces **all** the existing members. If there really is a padding somewhere that's only on one platform, then this should have a special case that pushes a 0-length type to the struct with the information about the padding.

Reply:
> this should have a special case that pushes a 0-length type to the struct with the information about the padding.

Fixed, hopefully :)
For the import at least